card/mod/standard/set/all/links.rb

```diff
@@ -1,81 +1,93 @@
-format do
-  # link is called by web_link, card_link, and view_link
-  # (and is overridden in other formats)
-  def link_to text, pathish, _opts={}
-    href = interpret_pathish pathish
+RESOURCE_TYPE_REGEXP = /^([a-zA-Z][\-+\.a-zA-Z\d]*):/
 
+format :html do
+  def link_to text=nil, opts={}
```

Comment:
what is the reasoning that we use a `path` key instead of the rails pattern with an `options` and a `html_options` hash?